stix_shifter/stix_transmission/src/modules/security_advisor/README.md

```diff
@@ -0,0 +1,39 @@
+#### Transmission - used to communicate with datasource (Security Advisor). Functionalities: query <query string>, status <search id>, results <search id> <offset> <length>, ping,
+
+Ping Security Advisor: 
+
+```
+python3 main.py transmit security_advisor  {} '{ "ibmCloudAccountID" :"", "ibmCloudApiKey" : "", "saAPIEndpoint" :"https://us-south.secadvisor.cloud.ibm.com/findings/v1/" }'  ping
```

Comment:
This isn't how the connection and configuration objects will be coming into stix-shifter. Would the cloud account id and api key change per user or does it it set for the data source?

Reply:
This per user ...